packages/curve-ui-kit/src/features/connect-wallet/ui/ConnectWalletPrompt.tsx

```diff
@@ -16,16 +24,62 @@ export const ConnectWalletPrompt = ({
   connectWallet,
   isLoading,
 }: ConnectWalletPromptProps) => (
-  <ConnectWallet
-    connectWallet={connectWallet}
-    description={description}
-    connectText={connectText}
-    loadingText={loadingText}
-    isLoading={isLoading}
-    theme={useUserProfileStore((state) => state.theme)}
-  />
+  <Stack
+    padding={7}
+    sx={{
+      backgroundColor: 'var(--table--background-color)',
+      maxWidth: '50rem',
```

Comment:
I know it's based on the old component but imo we should get rid of the magic numbers and use design system sizes and spaces